content/courses/native-onchain-development/serialize-instruction-data-frontend.md

```diff
@@ -269,7 +282,7 @@ submit a movie review and have it stored on Solana’s network. We’ll build th
 app a little bit at a time over the next few lessons, adding new functionality
 each lesson.
 
-![Movie review frontend](/public/assets/courses/unboxed/movie-reviews-frontend.png)
+![Movie review frontend](/public/assets/courses/superteam/movie-review-frontend-dapp.png)
```

Comment:
1. Include the image in this PR.
2. Unboxed originally made the courses, but honestly we don't want to be putting all the files into different organization directories (we should get rid of the unboxed folder too). Just use  `/public/assets/courses/movie-review-frontend.png`

Reply:
> 1. Include the image in this PR.
> 2. Unboxed originally made the courses, but honestly we don't want to be putting all the files into different organization directories (we should get rid of the unboxed folder too). Just use  `/public/assets/courses/movie-review-frontend.png`

fixed ✅ - check #346 - https://github.com/solana-foundation/developer-content/pull/346/commits/272fc38762a788c0b6b499c1127703c633705e29

Reply:
This PR! Not other PRs.

Reply:
done!